Web/starter_project_iconics/components/layout/Footer.tsx

```diff
@@ -0,0 +1,116 @@
+import Image from 'next/image'
+import {
+  AiFillLinkedin,
+  AiFillYoutube,
+  AiOutlineInstagram,
+  AiOutlineTwitter,
+} from 'react-icons/ai'
+import { MdCopyright, MdOutlineFacebook } from 'react-icons/md'
+
+const Footer: React.FC = () => {
+  const navigation = [
+    {
+      header: 'Links',
+      links: [
```

Comment:
I assume this navigation list is also useful in the navbar. it would be nice if we could share the same list for the two
i could recommend maybe exporting from one and importing in the other